main.py

```diff
@@ -258,6 +258,14 @@ async def ping(ctx):
     Checks if bot is alive
     '''
     await ctx.send('Hello world')
+# custome help commands
+@bot.command()
+async def cmds(ctx): # Help command
+    if check_admin ==True:
+    #if ctx.message.author.check_admin:
```

Comment:
administrator status not required to see list of commands, as its not sensitive information :)

Reply:
oh, I might have put check_admin by mistake, just ignore that